workflows-scripts/data-export/README.md

```diff
@@ -0,0 +1,43 @@
+## Workflow steps:
+
+##### Steps to generate data:
+1. Open Inventory App -> Instance tab -> select query and type for example: "source= marc" to get search records
+2. On top right select "Save Instance UUIDs"
+3. csv file will be downloaded
+4. Rename it and place it in data-export/jmeter-supported-data
+
+### Overview:
+#### Workflow for dataExport_exportMARCBibRecords.jmx
+##### Workflow in UI:
+1. Open Data export App
+2. On top left, choose .csv file with record IDS (browse to location where 10-instance-record-ids.csv exists)
+
+After specifying csv file, UI will return. Then, asynchronously POST data-export/export request will start a task  which calls mod-source-record-storage module
+
+##### Workflow in backend using JMeter: 
+1. `POST data-export/fileDefinitions` 
+- request: filename and size
+- response: returns filename and metadata
+2. `POST data-export/fileDefinitions/${fileDefinitionId}/upload`
+- request: file content as a Stream
+- response: file attributes json
+3. `POST data-export/export`
+- request: file definition and job profile
+- response: returns 204 on success and asynchronously makes a call to mod-source-record-storage
+4. Loop over `GET data-export/jobExecutions?query=id==${jobExecutionId}` endpoint until job status is SUCCESS otherwise return
```

Comment:
Is this how the UI determines the status of the job, by polling SRS? If so, what is the frequency that it polls? Should we call it with the same frequency that the UI polls?

Reply:
UI does not determine the status of the job. The backend does, by polling SRS. Because it is asynchronous.